### PR TITLE
Copy address to try a more internationalized interaction

### PR DESCRIPTION
**Pull request by cong1223**
Updates:

1. Update the copy address feature to incorporate a more internationalized interaction. For example, some leading web3 companies include openzeppelin.com 、consensys.io、etherscan.io and others.
2. Provide feedback in the connectwallet example after providing the link, as the Metamask pop-up won't appear when connecting again.

### CODE REVIEW

**Comment by cong1223** [comment]
I'm not sure how to fix this CI issue I'm facing. I can assure you that I have already passed the local test coverage and test CI. If someone could give me some hints, I would greatly appreciate it.

**Comment by kiner-tang** [comment]
image
You need to add test case to cover this line.

File: packages/web3/src/address/index.tsx [hunk 1]
```diff
@@ -57,28 +57,34 @@ export const Address: React.FC<React.PropsWithChildren<AddressProps>> = (props)
...
+        <>
+          {copied ? (
+            <CheckOutlined title="Address Copied!" />
+          ) : (
+            <CopyOutlined
+              title="Copy Address"
+              onClick={() => {
+                writeCopyText(filledAddress).then(() => {
+                  setCopied(true);
+                  setTimeout(() => {
```
Comment (kiner-tang) [inline packages/web3/src/address/index.tsx:79]: You should clear timeout before unmount

**Comment by cong1223** [comment]
> image You need to add test case to cover this line.

But why are all my tests passing when I run it locally?

**Comment by kiner-tang** [comment]
> > image You need to add test case to cover this line.
> 
> But why are all my tests passing when I run it locally?

The passing of the unit test can only indicate that the test results of the current existing test cases are not problematic, but this report shows that the test case coverage has decreased, that is, after you add new code, no new test cases are added to test your code. You can run the coverage test locally by running `pnpm test:coverage`. And view the `./coverage/index.html` test rate coverage report in your browser

**Comment by cong1223** [comment]
> > > image You need to add test case to cover this line.
> > 
> > 
> > But why are all my tests passing when I run it locally?
> 
> The passing of the unit test can only indicate that the test results of the current existing test cases are not problematic, but this report shows that the test case coverage has decreased, that is, after you add new code, no new test cases are added to test your code. You can run the coverage test locally by running `pnpm test:coverage`. And view the `./coverage/index.html` test rate coverage report in your browser

Thank you very much for your response. However, what confuses me is that the test coverage is also perfect when I run it locally.
image

**Comment by kiner-tang** [comment]
> > > > image You need to add test case to cover this line.
> > > 
> > > 
> > > But why are all my tests passing when I run it locally?
> > 
> > 
> > The passing of the unit test can only indicate that the test results of the current existing test cases are not problematic, but this report shows that the test case coverage has decreased, that is, after you add new code, no new test cases are added to test your code. You can run the coverage test locally by running `pnpm test:coverage`. And view the `./coverage/index.html` test rate coverage report in your browser
> 
> Thank you very much for your response. However, what confuses me is that the test coverage is also perfect when I run it locally. image

You can open `./coverage/index.html` in your browser and check this file change, you will get it.
image

**Comment by kiner-tang** [comment]
And you can see the report online: https://app.codecov.io/gh/ant-design/ant-design-web3/pull/448?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=ant-design

**Comment by cong1223** [comment]
> And you can see the report online: https://app.codecov.io/gh/ant-design/ant-design-web3/pull/448?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=ant-design

I peeped that, anyway thanks

**Review by kiner-tang — APPROVED** [review]

**Pull request merged**